README.md

```diff
@@ -2,12 +2,15 @@
 
 This is an open source emulator for [Azure Event Grid](https://learn.microsoft.com/en-us/azure/event-grid/overview) that supports only the [push delivery](https://learn.microsoft.com/en-us/azure/event-grid/push-delivery-overview) model. Based on ASP.NET Core, this emulator provides a cross-platform experience for developers wanting to try Azure Event Grid easily in a local environment without having to deploy to Azure.
 
+The emulator supports two Events delivery formats: Push model and Pull model.
```

Comment:
I think we should use `Push delivery` and `Pull delivery` everywhere to be consistent with the Azure taxonomy.

source: https://learn.microsoft.com/en-us/azure/event-grid/pull-delivery-overview